### PR TITLE
Add Font Optimization Example (Google Fonts)

### LINKED ISSUE #21: Add Google Fonts usage
https://nextjs.org/docs/basic-features/font-optimization

### PR DESCRIPTION
**Pull request by Jeonghyun-Kim**
branched from #27
-> it would be better to be merged after getting #27 merged.

resolves #21

### CODE REVIEW

File: src/pages/api/_document.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        <body>
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    );
+  }
+}
+
+export default MyDocument;
```
Comment (kjsik11) [inline src/pages/api/_document.tsx]: `src/pages/api` file path is wrong plz move to src/pages

**Review by kjsik11 — CHANGES_REQUESTED** [review]
change document file's path 🥲

File: src/pages/api/_document.tsx (hunk 1 above)
Comment (Jeonghyun-Kim) [reply src/pages/api/_document.tsx]: resolved by 8f81fbc

**Review by kjsik11 — APPROVED** [review]
goood

**Pull request merged**